packages/stryker-api/src/test_runner/RunOptions.ts

```diff
@@ -12,6 +12,8 @@ interface RunOptions {
    * It should be loaded right after the test framework but right before any tests can run.
    */
   testHooks?: string;
+
+  fileName?: string;
```

Comment:
Could we change this to `mutatedFileName`? This will further differentiate it from `fileNames` in the `RunnerOptions` interface.